src/App.js

```diff
@@ -1,23 +1,29 @@
-import logo from './logo.svg';
+import { useEffect, useState } from 'react';
 import './App.css';
 
+const COOL_NAMES = [
+  'james'
+]
+
 function App() {
+  const [name, setName] = useState('')
+  const [isCool, setIsCool] = useState(false)
+
+  useEffect(()=>{
+    setIsCool(
+      COOL_NAMES.includes(name.toLocaleLowerCase())
+      )
```

Comment:
```suggestion
    setIsCool(COOL_NAMES.includes(name.toLocaleLowerCase()))
```

please make this one line